packages/docs-page/index.js

```diff
@@ -111,7 +114,7 @@ export function DocsPageWrapper({
       </div>
       {/* if desired, show an "edit this page" link on the bottom right, linking to github */}
       {showEditPage && (
-        <div id="edit-this-page" className="g-container">
+        <div className={s.editThisPage}>
```

Comment:
Checked that [#edit-this-page is not being targeted in any of our projects](https://sourcegraph.hashi-mktg.com/search?q=%23edit-this-page&patternType=literal), so was fine to remove.